redshift_sqlalchemy/models.py

```diff
@@ -0,0 +1,55 @@
+"""
+System Tables and Views
+
+Amazon Redshift has many system tables and views that contain information
+about how the system is functioning. This module contains SQLAlchemy
+Declarative models for these tables.
+"""
+
+
+import sqlalchemy as sa
+from sqlalchemy.ext import declarative
+
+
+Base = declarative.declarative_base()
+
+
+class STLUnloadLog(Base):
```

Comment:
I'd strongly prefer this be named `StlUnloadLog`. Capitalizing just the first letter of abbreviations is the most consistent way to render names as CamelCase.